Documentation/PackedAssetBuilds.md

```diff
@@ -0,0 +1,139 @@
+```
+
+The outputs in windows_mono\bin\profile can be copied and run anywhere, once the pak files are put in the proper location.
+
+
+### Bundle Content
+
+Run the  AssetBundler
+
+```
+build\windows\bin\profile\AssetBundler.exe --project-path="c:\your\path\to\o3de-multiplayersample"
+```
+
+Follow steps for "Create a bundle for game assets", "Create a bundle for engine assets" and "Add bundles to the release game layout" from https://www.o3de.org/docs/user-guide/packaging/asset-bundler/bundle-assets-for-release/
+
+* The "default seed lists" choice should choose all but 4 seed lists to make the `engine_pc.pak`
+* The other 4 seed lists should all get selected to make the `game_pc.pak`
+
+> It's important to make sure that the bootstrap.game.profile.setreg file has been added to one of the seed files. (also add debug if you want to support debug builds)
+
+### Create the Launcher Zip file
+
+Use the following .bat file or equivalent copy steps to create a directory with the launchers in it:
+```shell
+rem Use this by calling 'make_release D:\my\output\dir' to make a release directory
+mkdir %1
+mkdir %1\Cache
+mkdir %1\Cache\pc
+mkdir %1\Gems
+mkdir %1\Gems\AWSCore
+ 
+rem Copy the pak files
+copy c:\your\path\to\o3de-multiplayersample\AssetBundling\Bundles\*.pak %1\Cache\pc
+ 
+rem Copy the executables and DLLs
+copy c:\your\path\to\o3de-multiplayersample\build\windows_mono\bin\profile\*.* %1
+ 
+rem Copy the AWSCore files?
+copy c:\your\path\to\o3de-multiplayersample\build\windows_mono\bin\profile\Gems\AWSCore\*.* %1\Gems\AWSCore
+ 
+rem Copy the launch_client / launch_server files
+copy c:\your\path\to\o3de-multiplayersample\launch_*.* %1
+ 
+rem Copy the PIX DLLs if PIX is enabled in the build
+:: copy D:\ly\lyengine\3rdParty\WinPixEventRuntime\bin\x64 %1
+```
+
+It's recommended during testing you track the commit IDs used from o3de, o3de-multiplayersample, and o3de-multiplayersample-assets to have as reference. 
+You can add this as .txt file in the build folder.
+
+Zip up the directory before running it to make sure the zip is "pure" without any logs or artifacts.
+
+### Run the final build to verify it
+
+```shell
+MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg --rhi=null -NullRenderer -bg_ConnectToAssetProcessor=0 -sys_PakPriority=2 -sv_terminateOnPlayerExit=false
+MultiplayerSample.GameLauncher.exe --console-command-file=launch_client.cfg -bg_ConnectToAssetProcessor=0 -sys_PakPriority=2
+```
+
+After running, check the output logs to verify there aren't any crashes, missing assets, etc. If any assets are missing, go back to the AssetBundler step, add the assets, and repeat.
+
+### Run the Launchers from Visual Studio
+Rename the o3de-multiplayersample/Cache/pc folder to pc-loose
+Create an o3de-multiplayersample/Cache/pc folder
+Copy the engine_pc.pak and game_pc.pak files into that folder
+Use the same command-line parameters as above
+
```

Comment:
This should probably be removed. It's definitely not required for building the pak builds, and these instructions are a little messy and incomplete for describing how to run the pak builds from VS.